### DISCUSSION TITLE
Why doesn't this magnet link find any peers / download any data?

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#2436] [Q&A]
![screen shot 2017-10-04 at 14 56 26](https://user-images.githubusercontent.com/10684786/31179622-5852bb02-a914-11e7-8107-1294fc9ffb57.png)

I just added a magnet link and for the past 20 minutes that is all I see.

I was getting annoyed that it wouldn't start I opened the same magnet link in qBittorrent and _instantly_ the download started (it's only been 2 minutes and it's already almost done)

Transmission > Preferences > Network shows port as closed. I don't know if that has anything to do with this problem

This is my first time using transmission

Transmission: 2.92
macOS High Sierra 10.13

**Comment by MoBattah** [comment]
It may very well be due to your port being closed. Please open it via port-forwarding, firewall rules, upnp, etc. 

Once you resolve that, tell me if the issue persists.

**Comment by romangb** [comment]
Hi, i have the same problem when trying to download magnet links, in despite that the port is forwarded.
![2017-11-29_23-00-47](https://user-images.githubusercontent.com/13413194/33399144-15470472-d55a-11e7-9663-3a19b63a3575.png)

**Comment by romangb** [comment]
seems that transmission can't connect to udp tracker address, i observe that in trackers window all trackers address are udp and all address have the status "Announce error: Connection failed"

**Comment by romangb** [comment]
Using a port from range  49152-65535 in "Peer listening port" solve the issue.

**Comment by joshuawprice** [comment]
Changing peer-port-random-low in settings.json also lets udp trackers connect in case you don't get a choice over which port you can use.
